.eslintrc.json

```diff
@@ -1,271 +1,312 @@
 {
-    "extends": "eslint:recommended",
```

Comment:
I think it would be convenient to not change the indenting in this PR so there's fewer rebase issues with my document/window PR, and so it's easier to review changes here.

Reply:
You have a good point there. Can you please update your PR to use 2 spaces instead of 4? That should minimize most of it. I'm not sure if/when, we'll be able to get that in as it doesn't have non-zero risk. I would rather not keep this file using inconsistent 4 space formatting for an extended period of time.

Reply:
> I'm not sure if/when, we'll be able to get that in as it doesn't have non-zero risk

as it has non-zero risk?

Reply:
I'd be happy with the indent change happening in this PR as long as it's in one atomic commit at the end

Reply:
It's currently impossible to review

Reply:
OK, I understand. That's the hard thing with the original formatting not being the same as the project. I guess it's using CLANG for formatting atm (which I don't actually use). I'm not sure why `.editorconfig` isn't used other than Google is used to CLANG. Either way, I don't have your original formatting settings. I'll set it back to 4 spaces and see how that looks.

Reply:
Pushed. Yeah, the diff looks much better now. Please take a look.